modules/gitops-release-notes-1-8-2.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-2_{context}"]
+= Release notes for {gitops-title} 1.8.2
+
+{gitops-title} 1.8.2 is now available on {product-title} 4.10, 4.11, and 4.12.
+
+[id="fixed-issues-1-8-2_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, when you configured Dex using the `.spec.dex` parameter and tried to log in to the Argo CD UI by using *Log In Via Openshift*, you were not able to log in. This update fixes the issue. 
++
+IMPORTANT: The `spec.dex` parameter in the ArgoCD CR is deprecated. In the future release of Red Hat Openshift GitOps v1.9, configuring Dex using the `spec.dex` parameter in the ArgoCD CR is planned to be removed. Consider using the `.spec.sso` parameter instead. See "Enabling or disabling Dex using .spec.sso".  link:https://issues.redhat.com/browse/GITOPS-2761[GITOPS-2761]
```

Comment:
See above for fixes to a phrase here, my suggested version is 

In a future release of {gitops-title} v1.9, configuring Dex using the `spec.dex` parameter in the ArgoCD CR is likely to be removed.

Reply:
Please see my previous comment regarding this comment.